ansible/roles/base/tasks/main.yml

```diff
@@ -10,6 +10,26 @@
 - name: Set DHCP config
   template: src=etc/dhcp/dhclient.conf.j2 dest=/etc/dhcp/dhclient.conf
 
+# Node
+- name: Register NodeJS
```

Comment:
Ideally these should be in a separate role. See https://github.com/TomAdam/ansible-roles/tree/master/node

Reply:
Moved into a `yarn` role.